### DISCUSSION TITLE
How to dynamically update the absolute target during a moveAsync?

### DISCUSSION THREAD

**Discussion by Gregory-Heck** [luni64/TeensyStep#162] [Q&A]
Hello,

I'm using the TeensyStep library and would like to dynamically update the motor's absolute target position while it is executing a movement with moveAsync. Is there a recommended way to modify the target position before the current movement is finished?

If possible, could you provide an example of how this can be implemented?

Thank you for your help!

**Comment by luni64** [comment]
Sorry it is not possible to change the target while moving with TeensyStep.
